docs/configuration/source/kubernetes.md

```diff
@@ -436,8 +436,8 @@ extraButtons:
     trigger:
       type: ["error"]
     button:
-      displayName: "Get Help"
-      commandTpl: "doctor --resource={{ .Kind | lower }}/{{ .Name }} --namespace={{ .Namespace }} --error={{ .Reason }} --bk-cmd-header='AI assistance'"
+      displayName: "Ask AI"
+      commandTpl: "ai why the {{ .Kind | lower }} '{{ .Name }}'  in the {{ .Namespace }} namespace is failing with {{ .Reason }}. If possible, check the application logs."
```

Comment:
AI plugin isn't available for self-hosted installations, and user doesn't configure the field here, so this should be just deleted. you can add a commented example for e.g. kubectl describe:

```yaml
# extraButtons:
#  - enabled: false
#   trigger:
#      type: ["error"]
#    button:
#    button:
#      displayName: "Describe"
#      commandTpl: "kubectl describe {{ Kind | lower }}{{ if .Namespace }} -n {{ .Namespace }}{{ end }} {{ .Name }}"
```